airflow/cli/commands/task_command.py

```diff
@@ -369,9 +369,9 @@ def task_run(args, dag=None):
                 dag = get_dag_by_deserialization(args.dag_id)
             except AirflowException:
                 print(f'DAG {args.dag_id} does not exist in the database, trying to parse the dag_file')
-                dag = get_dag(args.subdir, args.dag_id)
+                dag = get_dag(args.subdir, args.dag_id, include_examples=False)
         else:
-            dag = get_dag(args.subdir, args.dag_id)
+            dag = get_dag(args.subdir, args.dag_id, include_examples=False)  # --raw
```

Comment:
Do we need this comment?
```suggestion
            dag = get_dag(args.subdir, args.dag_id, include_examples=False)
```

Reply:
If we do, can we add it above this line with more detail

Reply:
We don't need it anymore